setup.py

```diff
@@ -1001,6 +1010,14 @@ def build_extensions(self) -> None:
         else:
             self._remove_extension("PIL._avif")
 
+        jpegxl = feature.get("jpegxl")
+        if isinstance(jpegxl, str):
+            # jxl and jxl_threads are required
```

Comment:
I would have thought that jxl_threads should be an optional extra, not a requirement? If it's only purpose is to speed things up, then if a user has jxl present on their system but not jxl_threads, then that doesn't sound like we should stop them from using the jxl library.